src/libraries/Microsoft.Extensions.DependencyModel/src/LibraryCollectionExtensions.cs

```diff
@@ -0,0 +1,28 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+namespace Microsoft.Extensions.DependencyModel
+{
+    using System;
+    using System.Collections.Generic;
+
+    internal static class LibraryCollectionExtensions
```

Comment:
This may make more sense as an internal method in https://github.com/dotnet/runtime/blob/main/src/libraries/Microsoft.Extensions.DependencyModel/src/CollectionExtensions.cs